test/config_parser_SUITE.erl

```diff
@@ -62,6 +77,10 @@ compare_values({auth_method, _}, V1, V2) when is_atom(V1) ->
     ?eq([V1], V2);
 compare_values({s2s_addr, _}, {_, _, _, _} = IP1, IP2) ->
     ?eq(inet:ntoa(IP1), IP2);
+compare_values(services, V1, V2) ->
+    V1_m = proplists:get_value(service_mongoose_system_metrics, V1),
+    V2_m = proplists:get_value(service_mongoose_system_metrics, V2),
+    compare_unordered_lists(V1_m, V2_m);
```

Comment:
I understand that using single letter variables names is tempting but please avoid it as it is much harder for someone else to read.
```suggestion
compare_values(services, Val1, Val2) ->
    MIMSysMetrics1 = proplists:get_value(service_mongoose_system_metrics, Val1),
    MIMSysMetrics2 = proplists:get_value(service_mongoose_system_metrics, Val2),
    compare_unordered_lists(MIMSysMetrics1, MIMSysMetrics2);
```

Reply:
I agree that this module could be more readable after refactoring, but I also think that such refactoring should be done in a separate story as it would fall out of scope here.